code/game/objects/items/clown_items.dm

```diff
@@ -246,3 +246,47 @@
 	desc = "Just looking at this makes you want to giggle."
 	icon_state = "laughter"
 	list_reagents = list(/datum/reagent/consumable/laughter = 50)
+
+/obj/item/rubber_chicken
+	name = "rubber chicken"
+	desc = " A robust rubber chicken the size of a Louisiana broiler.\nIt looks somewhat clammy and exudes a noticable rubbery smell.\nA small peeling sticker on the chicken's left thigh reads: 'made by waffle co.'"
+	icon = 'icons/obj/clown_items.dmi'
+	icon_state = "rubber_chicken"
+	force_string = "hilarious"
+	force = 10
+	damtype = STAMINA
+	hitsound = 'sound/weapons/genhit1.ogg'  //could use a funnier
+	attack_verb_continuous = list("slaps", "smacks", "gahonks")
+	attack_verb_simple = list("slap", "smack", "gahonk")
+
+/obj/item/rubber_chicken/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(!isliving(target))
+		return
+	var/mob/living/slapped = target
+	slapped.apply_status_effect(STATUS_EFFECT_PRANKED)
+
+/obj/item/rubber_chicken/attack_secondary(mob/living/victim, mob/living/user, params)
+	if(HAS_TRAIT(user, TRAIT_PACIFISM))
+		to_chat(user, "<span class='warning'>You are a pacifist.</span>")
```

Comment:
this item purely does stamina damage, so I don't see why pacifists aren't allowed to use it. pacifists can use other stamina damage weaponry, like stun batons, classic batons, and disablers.

Reply:
It is too cruel. I could be convinced otherwise though.